### PR TITLE
Fix stale bot

### PR DESCRIPTION
**Pull request by LysandreJik**
Fixes the stale bot

### CODE REVIEW

File: scripts/stale.py [hunk 1]
```diff
@@ -37,19 +37,20 @@ def main():
     open_issues = repo.get_issues(state="open")
 
     for issue in open_issues:
+        print(issue)
```
Comment (LysandreJik) [inline scripts/stale.py:40]: I'm leaving this print in in case we'd like to debug; it wasn't possible previously.

File: scripts/stale.py [hunk 2]
```diff
@@ -37,19 +37,20 @@ def main():
...
         comments = sorted([comment for comment in issue.get_comments()], key=lambda i: i.created_at, reverse=True)
         last_comment = comments[0] if len(comments) > 0 else None
         if (
             last_comment is not None and last_comment.user.login == "github-actions[bot]"
-            and (dt.utcnow() - issue.updated_at).days > 7
-            and (dt.utcnow() - issue.created_at).days >= 30
+            and (dt.utcnow() - issue.updated_at.replace(tzinfo=None)).days > 7
```
Comment (ydshieh) [inline scripts/stale.py:45]: so the cause is some API changes from GitHub?

**Review by ydshieh — APPROVED** [review]
Thank you for saving the poor bot's life!

File: scripts/stale.py (hunk 2 above)
Comment (LysandreJik) [reply scripts/stale.py:45]: Yes, I believe so

**Pull request merged**